setup.py

```diff
@@ -19,6 +19,7 @@
     license='Apache 2.0',
     install_requires=[
         'Jinja2==2.11.3',
+        'google-auth',
```

Comment:
This was a dependency now, and it got cut out with the last commit.  Not sure why, but it is definitely still needed.

Reply:
I removed it intentionally because it was already being installed as a dependency. See these logs, google-auth is being installed:
,
```
└─[$] <> pip install jinja2-gcp-secret-manager
Collecting jinja2-gcp-secret-manager
  Downloading jinja2_gcp_secret_manager-1.1.0-py3-none-any.whl (6.9 kB)
Collecting Jinja2==2.11.3
  Downloading Jinja2-2.11.3-py2.py3-none-any.whl (125 kB)
     |████████████████████████████████| 125 kB 4.0 MB/s 
Collecting proto-plus==1.13.0
  Downloading proto-plus-1.13.0.tar.gz (44 kB)
     |████████████████████████████████| 44 kB 2.7 MB/s 
Collecting google-api-core[grpc]==1.25.1
  Downloading google_api_core-1.25.1-py2.py3-none-any.whl (92 kB)
     |████████████████████████████████| 92 kB 333 kB/s 
Collecting google-cloud-secret-manager==2.2.0
  Downloading google_cloud_secret_manager-2.2.0-py2.py3-none-any.whl (90 kB)
     |████████████████████████████████| 90 kB 3.8 MB/s 
Collecting protobuf==3.14.0
  Downloading protobuf-3.14.0-py2.py3-none-any.whl (173 kB)
     |████████████████████████████████| 173 kB 10.8 MB/s 
Collecting googleapis-common-protos==1.52.0
  Downloading googleapis_common_protos-1.52.0-py2.py3-none-any.whl (100 kB)
     |████████████████████████████████| 100 kB 6.6 MB/s 
Collecting grpc-google-iam-v1==0.12.3
  Downloading grpc-google-iam-v1-0.12.3.tar.gz (13 kB)
Collecting libcst>=0.2.5
  Downloading libcst-0.3.16-py3-none-any.whl (505 kB)
     |████████████████████████████████| 505 kB 12.9 MB/s 
Collecting MarkupSafe>=0.23
  Downloading MarkupSafe-1.1.1-cp39-cp39-manylinux2010_x86_64.whl (32 kB)
Collecting requests<3.0.0dev,>=2.18.0
  Downloading requests-2.25.1-py2.py3-none-any.whl (61 kB)
     |████████████████████████████████| 61 kB 5.5 MB/s 
Collecting six>=1.13.0
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Collecting google-auth<2.0dev,>=1.21.1
  Downloading google_auth-1.24.0-py2.py3-none-any.whl (114 kB)
     |████████████████████████████████| 114 kB 7.0 MB/s 
Requirement already satisfied: setuptools>=40.3.0 in ./.virtualenvs/hello/lib/python3.9/site-packages (from google-api-core[grpc]==1.25.1->jinja2-gcp-secret-manager) (49.2.1)
Collecting pytz
  Downloading pytz-2021.1-py2.py3-none-any.whl (510 kB)
     |████████████████████████████████| 510 kB 3.5 MB/s 
Collecting grpcio<2.0dev,>=1.29.0; extra == "grpc"
  Downloading grpcio-1.35.0-cp39-cp39-manylinux2014_x86_64.whl (4.1 MB)
     |████████████████████████████████| 4.1 MB 8.2 MB/s 
Collecting typing-inspect>=0.4.0
  Downloading typing_inspect-0.6.0-py3-none-any.whl (8.1 kB)
Collecting pyyaml>=5.2
  Downloading PyYAML-5.4.1-cp39-cp39-manylinux1_x86_64.whl (630 kB)
     |████████████████████████████████| 630 kB 2.7 MB/s 
Collecting typing-extensions>=3.7.4.2
  Downloading typing_extensions-3.7.4.3-py3-none-any.whl (22 kB)
Collecting chardet<5,>=3.0.2
  Downloading chardet-4.0.0-py2.py3-none-any.whl (178 kB)
     |████████████████████████████████| 178 kB 4.5 MB/s 
Collecting idna<3,>=2.5
  Downloading idna-2.10-py2.py3-none-any.whl (58 kB)
     |████████████████████████████████| 58 kB 6.6 MB/s 
Collecting certifi>=2017.4.17
  Downloading certifi-2020.12.5-py2.py3-none-any.whl (147 kB)
     |████████████████████████████████| 147 kB 7.1 MB/s 
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.3-py2.py3-none-any.whl (137 kB)
     |████████████████████████████████| 137 kB 7.6 MB/s 
Collecting pyasn1-modules>=0.2.1
  Downloading pyasn1_modules-0.2.8-py2.py3-none-any.whl (155 kB)
     |████████████████████████████████| 155 kB 11.0 MB/s 
Collecting cachetools<5.0,>=2.0.0
  Downloading cachetools-4.2.1-py3-none-any.whl (12 kB)
Collecting rsa<5,>=3.1.4; python_version >= "3.6"
  Downloading rsa-4.7-py3-none-any.whl (34 kB)
Collecting mypy-extensions>=0.3.0
  Downloading mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
Collecting pyasn1<0.5.0,>=0.4.6
  Downloading pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
     |████████████████████████████████| 77 kB 4.1 MB/s 
Using legacy 'setup.py install' for proto-plus, since package 'wheel' is not installed.
Using legacy 'setup.py install' for grpc-google-iam-v1, since package 'wheel' is not installed.
Installing collected packages: MarkupSafe, Jinja2, six, protobuf, proto-plus, chardet, idna, certifi, urllib3, requests, pyasn1, pyasn1-modules, cachetools, rsa, google-auth, pytz, googleapis-common-protos, grpcio, google-api-core, typing-extensions, mypy-extensions, typing-inspect, pyyaml, libcst, grpc-google-iam-v1, google-cloud-secret-manager, jinja2-gcp-secret-manager
    Running setup.py install for proto-plus ... done
    Running setup.py install for grpc-google-iam-v1 ... done
Successfully installed Jinja2-2.11.3 MarkupSafe-1.1.1 cachetools-4.2.1 certifi-2020.12.5 chardet-4.0.0 google-api-core-1.25.1 google-auth-1.24.0 google-cloud-secret-manager-2.2.0 googleapis-common-protos-1.52.0 grpc-google-iam-v1-0.12.3 grpcio-1.35.0 idna-2.10 jinja2-gcp-secret-manager-1.1.0 libcst-0.3.16 mypy-extensions-0.4.3 proto-plus-1.13.0 protobuf-3.14.0 pyasn1-0.4.8 pyasn1-modules-0.2.8 pytz-2021.1 pyyaml-5.4.1 requests-2.25.1 rsa-4.7 six-1.15.0 typing-extensions-3.7.4.3 typing-inspect-0.6.0 urllib3-1.26.3
```

Reply:
👍🏼 Removed that dependency from `setup.py`